### PR TITLE
completed react-form-controls

### PR DESCRIPTION
**Pull request by m-Cho1**
image

### CODE REVIEW

File: react-form-controls/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        <form onSubmit={this.handleSubmit}>
+          <label>
+            Username
+            <input type="text" value={this.state.username} onChange={this.handleUsernameChange} />
+            </label>
+          <label>
+            Password
+            <input type="password" value={this.state.password} onChange={this.handlePasswordChange} />
+            </label>
+          <input type="submit" value='Sign Up' onChange={this.handleSubmit} />
```
Comment (scbowler) [inline react-form-controls/src/index.jsx:43]: No on change handler should be used on the submit button input. The `<form>`s `onSubmit` handler will handle that.

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, but one small update needed. Let me know if you have any questions.

**Comment by m-Cho1** [comment]
Edited submit input

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**